src/debug/debugSession.ts

```diff
@@ -454,7 +454,8 @@ export class DebugSession implements IDebugSession {
     private async openInBrowser(errorMessage: string, link: string): Promise<void> {
         const answer = await vscode.window.showErrorMessage(errorMessage, "Open in Browser");
         if (answer === "Open in Browser") {
-            opn(link);
+            const openExternal: any = (<any>vscode.env).open ? (<any>vscode.env).open : opn;

```

Comment:
Oh, it's used more than once - definitely move it to a module then!  *grin*

Reply:
done.